presto-main/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -449,12 +450,27 @@ public int getHttpTimeoutThreads()
     }
 
     @Config("task.http-timeout-threads")
+    @ConfigDescription("Total number of timeout threads")
     public TaskManagerConfig setHttpTimeoutThreads(int httpTimeoutThreads)
     {
         this.httpTimeoutThreads = httpTimeoutThreads;
         return this;
     }
 
+    @Min(1)
+    public int getHttpTimeoutConcurrency()
+    {
+        return httpTimeoutConcurrency;
+    }
+
+    @Config("task.http-timeout-concurrency")
+    @ConfigDescription("Number of thread pool to divide threads configured by property task.http-timeout-threads")
```

Comment:
Maybe

`The estimated number of timeout handlers being attached concurrently`? Or something similar?

Basically in the description we should express that those two properties are different, and are used to configure different concurrency level.

- `task.http-timeout-threads` - number of threads that are processing timeout handlers
- `task.http-timeout-concurrency` - estimated number of threads (or estimated thread congestion) when attaching timeout handlers

Reply:
How about ``The maximal number of http request timeout handlers being attached concurrently`` ?

Reply:
@aweisberg @wenleix ?

Reply:
`Number of thread pools to handle timeouts. Threads per pool is http-timeout-threads / http-timeout-concurrency`

Reply:
Sounds good to me :).  I think these two configs are inherently complicated (inherited from airlift) so we probably just want to make sure it's well documented .